### DISCUSSION TITLE
Azimuthal orientation on Qfield 3.4.4 ist not properly adjusted when devise is rotated

### DISCUSSION THREAD

**Discussion by ranleu** [opengisch/QField#5820] [Bugs]
When the field devise is rotated while using QField the azimuthal orientation (@position_orientation)  does not adjust properly. If the tablet is looking exactly to the East I expect the Orientation to always be 90 degrees no matter if my device is rotated to landscape or to vertical. I also expect it to show the proper orientation irrespective of its tilt with respect to the zenith.

However, it adds/substracts 90 degrees to the azimuth while rotating even recording values above 360 degrees.

If I make my device "go to sleep", turn it and then "wake it" and activate QField again it adjusts the orientation properly but this does not happen "on the fly" when QField is active. 

This issue appears on iPads 10th generation with iPadOS 17.3. 
I will test the issue on Android systems and update the discussion with the results.

**Comment by nirvn** [comment]
@ranleu , please do check with Android and report whether it's an iOS-specific issue or not.

**Reply by ranleu** [reply]
I have checked on the following:

- iPhone with iOS 18.1.1 --> azimuthal orientation adjusts on the fly when rotating. When rotated to vertical, all fine. When rotated to horizontal, orientation values between North and East are shown as >360 and <450. Beyond East orientation values are correctly displayed again. 

- Huawei Phone with EMUI 12.0.0 --> azimuthal orientation adjusts on the fly when rotating and is correct both in vertical and horizontal rotation.

I am expecting to try it on a purely Android device within the next couple of days. Update: the Android device I have access to, seems to be too old and its OS is not able to handle newer QField versions.

My conclusion so far: the problem seems to be related wit the iOS versions and while there is still a bug on iPhones the problem is stronger on iPads.

**Comment by ranleu** [comment]
Please reactivate the report. I did reply to it.

**Reply by nirvn** [reply]
@ranleu , which report?

**Reply by ranleu** [reply]
This one and #5819. I am not sure if "report" ist the right terminology.
